### PR TITLE
CNV-35286: fix bandwidthPerMigration documentation

### PR DESCRIPTION
**Pull request by tiraboschi**
Version(s):
v4.11+

Issue:
https://issues.redhat.com/browse/CNV-35286

Link to docs preview:
https://69254--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/live_migration/virt-configuring-live-migration#virt-configuring-live-migration-limits_virt-configuring-live-migration

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by tiraboschi** [comment]
@apinnick FYI

File: modules/virt-configuring-live-migration-limits.adoc [hunk 1]
```diff
@@ -35,7 +35,7 @@ spec:
     parallelOutboundMigrationsPerNode: 2 <4>
     progressTimeout: 150 <5>
 ----
-<1> Bandwidth limit of each migration, in MiB/s or GiB/s. Default: `0`, which is unlimited.
+<1> Bandwidth limit of each migration, the value is quantity of bytes per second (e.g. 2048Mi = 2048MiB/sec). Default: `0`, which is unlimited.
```
Comment (ousleyp) [inline modules/virt-configuring-live-migration-limits.adoc:38]: Suggestion for clarity and improved adherence to style guides :) 

```
<1> Bandwidth limit of each migration, where the value is the quantity of bytes per second. For example, a value of `2048Mi` means 2048 MiB/s. Default: `0`, which is unlimited.
```

**Review by ousleyp — COMMENTED** [review]
This is close, but I suggested a change that will make it comply with our style rules. After you make that change and squash your commits down to 1 commit, please tag me and I will merge this to 4.11+. Thanks, @tiraboschi :)

**Comment by ousleyp** [comment]
Thanks, @tiraboschi! LGTM. Merging now.

**Comment by ousleyp** [comment]
I manually cherrypicked this to 4.13 by adding it to the module where this information was in 4.13. Cherrypicking https://github.com/openshift/openshift-docs/pull/69985 to 4.11 and 4.12 now.

**Pull request merged**